client/blocks/nav-unification-modal/index.jsx

```diff
@@ -116,7 +116,12 @@ const Modal = () => {
 							}
 							heading={ translate( 'Do even more' ) }
 							content={ translate(
-								'Advanced admin features have a new home! You can find it in your account settings'
+								'Advanced admin features have a new home! You can find it in your {{a}}Account Settings{{/a}}.',
```

Comment:
Before merging, could I just ask if this should be changed? I just spotted it now, but I feel it reads a little better. 

```suggestion
								'Advanced admin features have a new home! You can find them in your {{a}}Account Settings{{/a}}.',
```

Reply:
That seems like a good update to make. If you are able to push the commit I'll re-run the tests again. Thanks for spotting that.

Reply:
Thanks! I updated it.